control-plane/stor-port/src/types/v0/transport/volume.rs

```diff
@@ -717,12 +717,15 @@ pub struct SetVolumeProperty {
     /// The uuid of the volume.
     pub uuid: VolumeId,
     /// The property to set.
-    pub property: VolumeProperty,
+    pub property: Option<VolumeProperty>,
```

Comment:
Should not be optional, if required we can ensure  grpc conversions error out if None

Reply:
fixed.